### PR TITLE
fix: Well-Known Annotations/Labels/Taints examples

### PR DESCRIPTION
**Pull request by jai**
Fixes https://github.com/kubernetes/website/issues/32989

### CODE REVIEW

**Comment by jai** [comment]
Example page: https://deploy-preview-32991--kubernetes-io-main-staging.netlify.app/docs/reference/labels-annotations-taints/

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassN
 
 ### storageclass.kubernetes.io/is-default-class
 
-Example: `storageclass.kubernetes.io/is-default-class=true`
+Example: `storageclass.kubernetes.io/is-default-class: true`
```
Comment (divya-mohan0209) [inline content/en/docs/reference/labels-annotations-taints/_index.md:352]:
```suggestion
Example: `storageclass.kubernetes.io/is-default-class: "true"`
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Small nit, otherwise lgtm.

**Pull request merged**